copter/source/docs/safety-multicopter.rst

```diff
@@ -196,27 +196,27 @@ MultiCopter Safety
    #. Gusting winds around objects can also worsen the problem.
    #. Radio signal loss can also occur.
    #. If your copter is approaching a potentially interfering object,
-      immediately switch to stabilize mode and land or retrieve the
+      immediately switch to Stabilize mode and land or retrieve the

```

Comment:
This makes it consistent, I think.

I agree, a future patch should canonicalise all instances to the uc version.